IDE/IAR-EWARM/Projects/user_settings.h

```diff
@@ -17,10 +17,35 @@
 
 #define WOLFSSL_GENSEED_FORTEST /* Warning: define your own seed gen */
 
-#define TFM_TIMING_RESISTANT
+/* A few examples of different math options below.
+ *
+ * See examples/configs/user_settings_template.h for a more
+ * detailed template. */
+#if 1
+    /* use SP math for all key sizes and curves */
+    #define WOLFSSL_SP_MATH_ALL
+    /* Disable use of dynamic stack items */
+    #define WOLFSSL_SP_NO_DYN_STACK
+    /* use smaller version of code */
+    #define WOLFSSL_SP_SMALL
+#elif 1
```

Comment:
I'd like to see a fourth option that enables the single precision math:
 
```
#define WOLFSSL_HAVE_SP_ECC
#define WOLFSSL_HAVE_SP_RSA
#define WOLFSSL_HAVE_SP_DH
#define WOLFSSL_SP_SMALL_STACK
#define WOLFSSL_SP_MATH /* only SP sp_c32.c implementation */
#define WOLFSSL_SP_SMALL
//#define WOLFSSL_SP_ARM_CORTEX_M_ASM /* Assembly optimized version - sp_cortexm.c */
```

Reply:
Done! I rearranged the comments a bit to keep the column width down.